tests/test_trainer.py

```diff
@@ -136,7 +135,7 @@ def test_grid_regression(self):
             CnnRegression,
             dataset
         )
-        trainer.train(nepoch=1, batch_size=2, save_best_model=None)
+        trainer.train(nepoch=1, batch_size=2, best_model=False, save_model=False)
 
```

Comment:
throughout this file, whenever `save_model=False`, you don't need to specify a `best_model` argument.

Reply:
Same as above, it does change which model is going to be used for later testing. Even if it's true that in the sake of this test, it doesn't matter because we're not testing last vs best. I placed filename=None in the tests where we're not interested in the saved model.